### PR TITLE
Fix to not highlight Tutorials, when navigating to them from the navigator links, then going back.

### PR DESCRIPTION
**Pull request by dobromir-hristov**
Bug/issue #, if applicable: 90840717

## Summary

This fix prevents keeping tutorials highlighted, when clicking on them in the Navigator, then going back.

This was caused because we kept an activeUID for that item upon clicking, but it navigates you to a different page type altogether. 

The fix is a more generic one, to clear the cached state, if the activeUID path is NOT the same as the current route path.

**Alternative**
An alt fix could be to not emit a click event for Tutorial types, but that is a slippery slope I think.

## Dependencies

NA

## Testing

Steps:
1. Open an archive with a tutorial
2. Click on a none-tutorial item
3. Then click on a tutorial
4. Assert you get navigated to the tutorial
5. Go back with browser Back
6. Assert the tutorial is no longer highlighted, but the current page is.

## Checklist

Make sure you check off the following items. If they cannot be completed, provide a reason.

- [x] Added tests
- [x] Ran `npm test`, and it succeeded
- [x] Updated documentation if necessary

### CODE REVIEW

File: src/components/Navigator/NavigatorCard.vue [hunk 1]
```diff
@@ -104,6 +104,7 @@ import { TopicTypes } from 'docc-render/constants/TopicTypes';
 import FilterInput from 'docc-render/components/Filter/FilterInput.vue';
 import { BreakpointName } from 'docc-render/utils/breakpoints';
 import keyboardNavigation from 'docc-render/mixins/keyboardNavigation';
+import { last } from '@/utils/arrays';
```
Comment (mportiz08) [inline src/components/Navigator/NavigatorCard.vue:107]:
```suggestion
import { last } from 'docc-render/utils/arrays';
```

File: src/utils/arrays.js [hunk 2]
```diff
@@ -0,0 +1,2 @@
+// eslint-disable-next-line import/prefer-default-export
+export const last = array => array[array.length - 1];
```
Comment (mportiz08) [inline src/utils/arrays.js:2]: I realize it's very simple, but we should add unit tests for this if we're going to add this helper. Also, this new file will need the Swift open source license preamble.

File: app/public/theme-settings.json [hunk 3]
```diff
@@ -58,7 +58,7 @@
         "hide": false
       },
       "navigator": {
-        "enable": false
```
Comment (marinaaisa) [inline app/public/theme-settings.json:61]: I don't think this change was intended for this PR :)

**Review by marinaaisa — APPROVED** [review]
Looks good! I left a small comment, thanks Dobri! ✨

**Comment by mportiz08** [comment]
@swift-ci test

**Pull request merged**